f/frizzle/comapeo/comapeo_observations.py

```diff
@@ -282,26 +316,13 @@ def handle_output(self, outputs):
         """
```

Comment:
so this method now writes to multiple tables?

would be good to write that in the docstring.